core-bundle/contao/templates/twig/backend/widget/password.html.twig

```diff
@@ -0,0 +1,35 @@
+{% trans_default_domain "contao_default" %}
+
+<div data-controller="contao--password-visibility">
+    {% if prependLabel|default %}
+        <label for="{{ id }}">{{ 'MSC.password.0'|trans }}</label>
+    {% endif %}
+    {% set input_attributes = attrs()
+        .set('type', 'password')
+        .set('name', name)
+        .set('id', id)
+        .addClass(['tl_text', 'tl_password', class])
+        .set('value', '')
+        .set('placeholder', value)
+        .set('autocomplete', 'new-password')
+        .set('data-contao--password-visibility-target', 'input')
+        .set('data-action', 'focus->contao--scroll-offset#store')
+        .mergeWith(attributes)
+    %}
+    <input{{ input_attributes }}>
+
+    {% set button_attributes = attrs()
+        .set('type', 'button')
+        .addClass('password-button')
+        .set('data-action', 'contao--password-visibility#toggle')
+    %}
+
+    <button{{ button_attributes}}>
+        <span data-contao--password-visibility-target="icon">{{ backend_icon('password--hidden.svg', 'MSC.showPassword'|trans) }}</span>
+        <span class="hidden" data-contao--password-visibility-target="icon">{{ backend_icon('password--visible.svg', 'MSC.hidePassword'|trans) }}</span>
+    </button>
+</div>
+
+{% if description|default %}
+    <p class="tl_help tl_tip">{{ description }}</p>
```

Comment:
I'm not sure this is correct. Our widgets do not add the description themselves, that is usually done by DC_Table when rendering the widgets. This would be a massive behavior change imho.

Reply:
This is how it has been in the version pror to rewriting it to twig.

The issue was us removing the description in the DataContainer which dates back further than 2012 (git blame doesn't show further)

See 528dac0c43ee8eb1427aedddda7b55127af1ab60